main.go

```diff
@@ -111,6 +124,9 @@ func checkArgs(_ *corev2.Event) error {
 	if icon := os.Getenv("SENSU_SLACK_ICON_URL"); icon != "" && config.slackIconURL == defaultIconURL {
 		config.slackIconURL = icon
 	}
+	if alert := os.Getenv("SENSU_SLACK_ALERT_CRITICAL"); alertCritical != "" && !config.slackAlertCritical {
```

Comment:
The manual environment variable lookups are technical debt and this code should probably be deleted at some point. No need to add this manual lookup, environment variables are handled transparently by the plugin SDK.

Reply:
Sure I have removed this in the latest push.  Thanks!